extension/sqlsmith/statement_generator.cpp

```diff
@@ -73,13 +88,29 @@ std::shared_ptr<GeneratorContext> StatementGenerator::GetDatabaseState(ClientCon
 			result->tables_and_views.push_back(entry);
 		});
 	}
+	if (context.transaction.HasActiveTransaction()) {
+		context.transaction.Commit();
+	}
 	return result;
 }
 
 unique_ptr<SQLStatement> StatementGenerator::GenerateStatement() {
-	if (RandomPercentage(80)) {
+	if (RandomPercentage(50)) {
 		return GenerateStatement(StatementType::SELECT_STATEMENT);
 	}
+	if (RandomPercentage(40)) {
```

Comment:
Maybe something like 
```
if (RandomPercentage(20)) {
    return GenerateAttachAndUse();
}
```
? I think it reads nicer if here it's immediately an attach and use instead of a MultiStatement